charts/backend/templates/deployment.yaml

```diff
@@ -101,18 +101,38 @@ spec:
 {{- if ne (len .Values.extraVolumeMounts) 0 }}
 {{ toYaml .Values.extraVolumeMounts | indent 12 }}
 {{- end }}
+
+{{ if .Values.probes.backend }}
+          livenessProbe:
+            httpGet:
+              path: /status/health
+              port: http
+            initialDelaySeconds: {{ if .Values.probes.backend.livenessProbe.initialDelaySeconds }}{{ .Values.probes.backend.livenessProbe.initialDelaySeconds }}{{ else }}30{{ end }}
+            periodSeconds: {{ if .Values.probes.backend.livenessProbe.periodSeconds }}{{ .Values.probes.backend.livenessProbe.periodSeconds }}{{ else }}10{{ end }}
+            timeoutSeconds: {{ if .Values.probes.backend.livenessProbe.timeoutSeconds }}{{ .Values.probes.backend.livenessProbe.timeoutSeconds }}{{ else }}1{{ end }}
+          readinessProbe:
+            httpGet:
+              path: /status/health
+              port: http
+            initialDelaySeconds: {{ if .Values.probes.backend.readinessProbe.initialDelaySeconds }}{{ .Values.probes.backend.readinessProbe.initialDelaySeconds }}{{ else }}1{{ end}}
+            periodSeconds: {{ if .Values.probes.backend.readinessProbe.initialDelaySeconds }}{{ .Values.probes.backend.readinessProbe.periodSeconds }}{{ else }}10{{ end }}
+            timeoutSeconds: {{ if .Values.probes.backend.readinessProbe.timeoutSeconds }}{{ .Values.probes.backend.readinessProbe.timeoutSeconds }}{{ else }}1{{ end }}
+{{ else }}
```

Comment:
I would suggest to remove the contents of this else block.

Reply:
done